docs/src/workflows.md

```diff
@@ -181,3 +181,42 @@ This will give you the ability to preview an animation with a slider:
 If you don't want to use the interactivity and simply display the output, set the `liveviewer` flag as `false` in the Javis `render` function and Pluto will display the gif:
 
 ![](assets/pluto_viewer.gif)
+
+
+## Livestreaming animations
+Javis brings in the support to pipe the rendered animations to streaming softwares(like OBS, VLC) or directly to platforms like Twitch or any alternatives that can read from a network stream.
+
+- ### Streaming Locally
+Many eaducational streamers/teachers prefer using streaming tools like OBS/VLC to have more cretive control over thier streams. Javis livestreaming is tuned for that use case.
+The animation can be streamed to a local IP and port and can be viewed using any tool that can read from a network stream.
+
+Firstly, one needs to setup the `StreamConfig` object that holds configuration regarding the streaming process. Javis uses `udp://127.0.0.1:8081` as the default streaming protocol, address and port respectively. However you're free to use any options of your choice.
```

Comment:
make sure to update this if you change the ip, port as this currently says 127.0.0.1